Examples/CMakeLists.txt

```diff
@@ -1 +1,73 @@
-add_subdirectory (CppClient)
+set (
+  CARLA_EXAMPLE_CPP_CLIENT_SOURCE_PATH
+  ${CARLA_WORKSPACE_PATH}/Examples/CppClient
+)
+
+set (
```

Comment:
Do we know if only passing carla-client as a dependency is enough? In theory carla-client should already be including all of these.